blockbase/sass/blocks/_post-comments.scss

```diff
@@ -12,14 +12,52 @@
 		@include button-hover-styles;
 	}
 
-	.reply a {
-		display: inline-block;
+	.reply {
+		margin-block: var(--wp--custom--margin--vertical);
+
+		a {
+			display: inline-block;
+		}
 	}
 
-	.comment-form {
+	form {
 		input[type="text"] {
 			width: 100%;
 		}
+
+		input:not([type=submit]):not([type=checkbox]),
+		textarea {
+			border: var(--wp--custom--form--border--width) var(--wp--custom--form--border--style) var(--wp--custom--form--border--color);
+		}
+
+		.comment-form-cookies-consent {
+			input[type="checkbox"] {
+				margin-left: 0;
+				
+				& + ::before,
+				& + ::after {
+					box-sizing: border-box;
```

Comment:
These are working for Quadrat but not for Blockbase because of the missing `position:relative;`. I think they need to be `& + label::before` and we need to add `position:relative;` to `& + label`